frame/society/src/lib.rs

```diff
@@ -788,7 +788,7 @@ decl_module! {
 			let mut payouts = <Payouts<T, I>>::get(&who);
 			if let Some((when, amount)) = payouts.first() {
 				if when <= &<system::Module<T>>::block_number() {
-					T::Currency::transfer(&Self::payouts(), &who, *amount, KeepAlive)?;
+					T::Currency::transfer(&Self::payouts(), &who, *amount, AllowDeath)?;
```

Comment:
@gavofyork: ping on this change

The `payouts` transfer allow death in all the other parts of the code. I assume this was just missed.

A user _should_ be able to get paid out using the full funds of the `payouts` account.

Reply:
This fixes a test which was failing now that ED is non-zero, and we weren't allowing a full payout.